src/command.rs

```diff
@@ -28,67 +41,100 @@ pub extern "C" fn wgpu_command_encoder_copy_buffer_to_buffer(
         destination,
         destination_offset,
         size))
+    .expect("Unable to copy buffer to buffer")
+}
+
+#[repr(C)]
+#[derive(Clone)]
+pub struct BufferCopyViewC {
+    pub layout: wgt::TextureDataLayout,
+    pub buffer: id::BufferId,
+}
+
+impl Into<wgc::command::BufferCopyView> for BufferCopyViewC {
```

Comment:
it would be simpler to just have this as `pub fn to_wgpu(&self) -> wgc::command::BufferCopyView`. since it would turn `xxx.cloned().into()` into `xxx.to_wgpu()`, which is much cleaner